lib/components/Modal/Modal.tsx

```diff
@@ -25,23 +26,83 @@ export const Modal = ({ open, onClose, content, progress, sidePanel, footer }: M
       }}
       className="ds-relative ds-z-50"
     >
+      {/* Backdrop */}
       <div className="ds-fixed ds-inset-0 ds-bg-black/30" aria-hidden />
-      <div className="ds-fixed ds-inset-0 ds-w-screen ds-py-6 sm:ds-py-[96px]">
-        <div className="ds-flex ds-h-full ds-items-center ds-justify-center">
-          <DialogPanel className="ds-flex ds-h-full ds-flex-col ds-justify-between ds-overflow-hidden ds-rounded sm:ds-rounded-lg ds-bg-bg-gray ds-min-w-full sm:ds-min-w-[640px]">
-            <div className="ds-my-6 ds-grid ds-max-h-[calc(100vh-172px)] ds-max-w-[1092px] ds-grid-cols-2 ds-gap-6 ds-pl-5 sm:ds-max-h-[calc(100vh-320px)] sm:ds-grid-cols-6 sm:ds-pl-9">
-              <div className="ds-col-span-2 ds-flex ds-max-h-[calc(100vh-172px)] ds-flex-col ds-gap-6 ds-pr-5 sm:ds-col-span-4 sm:ds-max-h-[calc(100vh-320px)] sm:ds-pr-0">
+      {/* Wrapper container paddings and margins */}
+      <div className="ds-fixed ds-inset-0 ds-py-6 sm:ds-py-[96px] ds-mx-3">
+        {/* Wrapper for container centering */}
+        <div className="ds-flex ds-items-center ds-justify-center">
+          {/* Modal container */}
+          <DialogPanel
+            className={tc(`
+            ds-flex
+            ds-flex-col
+          ds-bg-bg-gray
+            ds-justify-between
+            ds-overflow-hidden
+            ds-rounded
+            sm:ds-rounded-lg
+            sm:ds-min-w-[640px]
+          `)}
+          >
+            {/* Content wrapper */}
+            <div
+              className={tc(`
+              ds-my-6
+              ds-grid
+              ds-max-h-[calc(100vh-172px)]
+              ds-max-w-[1092px]
+              ds-grid-cols-2
+              ds-gap-6
+              ds-pl-5
+              sm:ds-max-h-[calc(100vh-320px)]
+              sm:ds-grid-cols-6
+              sm:ds-pl-9
+            `)}
+            >
+              {/* Main content */}
+              <div
+                className={tc(`
+                ds-col-span-2
+                ds-flex
+                ds-flex-col
+                ds-gap-y-6
```

Comment:
Changed from `ds-gap-6` to `ds-gap-y-6` because nothing happend with vertical positioning